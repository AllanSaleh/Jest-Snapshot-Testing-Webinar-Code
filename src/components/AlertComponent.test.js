import React from "react"
import renderer from 'react-test-renderer'
import AlertComponent from "./AlertComponent"

test("properly renders the Alert Component", ()=>{
    const tree = renderer.create(<AlertComponent />).toJSON();
    expect(tree).toMatchSnapshot();
})