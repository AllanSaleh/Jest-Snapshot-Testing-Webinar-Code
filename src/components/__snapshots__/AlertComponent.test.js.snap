// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`properly renders the Alert Component 1`] = `
<div
  className="fade alert alert-success show"
  role="alert"
>
  <div
    className="alert-heading h4"
  >
    Hey, nice to see you
  </div>
  <p>
    Allan Aww yeah, you successfully read this important alert message. This example text is going to run a bit longer so that you can see how spacing within an alert works with this kind of content.
  </p>
  <hr />
  <p
    className="mb-0"
  >
    Whenever you need to, be sure to use margin utilities to keep things nice and tidy.
  </p>
</div>
`;
